councilmatic/settings.py

```diff
@@ -104,12 +97,40 @@
 
 USE_ETAGS = True
 
-
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.8/howto/static-files/
-
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 
-
 HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
+
+# Uncomment for HTTPS sites
```

Comment:
What's going on here?

Reply:
Don't remember specifically. This has been in the example settings file for a while. I think it had something to do with figuring out how to reduce the number of redirects when going from HTTP to HTTPS, though.

Reply:
Why is it only on the new side of the diff?